src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -183,30 +194,31 @@ public boolean validCheckForObject(final AtlasObject object)
     private boolean validEdge(final AtlasObject object)
     {
         return object instanceof Edge
-                // Ignore any airport taxiways and runways, as these often create a sink island
-                && !Validators.isOfType(object, AerowayTag.class, AerowayTag.TAXIWAY,
-                        AerowayTag.RUNWAY)
-                // Only allow car navigable highways and ignore ferries
-                && HighwayTag.isCarNavigableHighway(object) && !RouteTag.isFerry(object)
+                // Only allow car navigable highways (access = yes and
+                // motorvehicle/motorcar/vehicle = yes)
+                // and ignore ferries
+                && HighwayTag.isCarNavigableHighway(object)
+                && Validators.isOfType(object, AccessTag.class, AccessTag.YES)
```

Comment:
I'm not following why we added this. My first thought is this is way too strict and will drastically limit the number of flags. Am I missing something?